src/applications/disability-benefits/all-claims/tests/cypress.helpers.js

```diff
@@ -254,10 +250,12 @@ export const pageHooks = (cy, testOptions = {}) => ({
   introduction: () => {
     cy.get('@testData').then(data => {
       if (data['view:isBddData']) {
-        window.sessionStorage.setItem(
-          SAVED_SEPARATION_DATE,
-          todayPlus120.join('-'),
+        const separationDate = format(
+          add(new Date(), { days: 120 }),
+          'yyyy-MM-dd',
```

Comment:
using same format here that is used in the [component](https://github.com/department-of-veterans-affairs/vets-website/blob/dbex/85579-TE-review-alert/src/applications/disability-benefits/wizard/pages/bdd.jsx#L13)